src/screens/Shared/EditLocation.js

```diff
@@ -45,15 +45,15 @@ class EditLocation extends Component {
       const results = await Geocoder.geocodeAddress(address);
       this.setState(
         {
-          results: results.map(result => {
+          results: results.map((result, index) => {
             const { lat, lng } = result.position;
             return {
               address_1: result.feature,
               city: result.locality,
               latitude: lat,
               coordinate: { latitude: lat, longitude: lng },
               longitude: lng,
-              identifier: `{lat}{lng}`,
+              identifier: `${index}`,
```

Comment:
If index is just a number, is string interpolation still necessary for the identifier?